core/spec/models/spree/concerns/user_address_book_spec.rb

```diff
@@ -167,9 +239,23 @@ module Spree
           expect(subject).to eq address
         end
 
-        it "sets it as default" do
-          subject
-          expect(user.default_address).to eq address
+        context "sets it as default" do
+          context "shipping address" do
+            it do
+              subject
+              expect(user.ship_address).to eq address
+              expect(user.default_address).to eq address
+            end
+          end
+
+          context "billing address" do
+            subject { user.save_in_address_book(address.attributes, true, :billing) }
+
+            it do
```

Comment:
Same as for the other: can you add a description here?

Reply:
Done (long ago)